docs/en/fleet-developer/fleet-architecture.asciidoc

```diff
@@ -5,6 +5,10 @@ beta[]
 
 coming[a future update]
 
+## Elasticsarch as communication layer
```

Comment:
`##` is markdown. Asciidocctor is lenient, but you should use `==` to be consistent with the rest of the docs.

Reply:
changed.